api/src/main/java/io/strimzi/api/kafka/model/template/KafkaExporterTemplate.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.template;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.UnknownPropertyPreserving;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Representation of a template for Kafka Exporter resources.
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "deployment", "pod", "podDisruptionBudget"})
+@EqualsAndHashCode
+public class KafkaExporterTemplate implements Serializable, UnknownPropertyPreserving {
+    private static final long serialVersionUID = 1L;
+
+    private ResourceTemplate deployment;
+    private PodTemplate pod;
+    private PodDisruptionBudgetTemplate podDisruptionBudget;
```

Comment:
Will you create PodDisruptionBudget for this? I'm not sure it really makes sense here. I do not think we create one for EO either.